hathor/version.py

```diff
@@ -12,4 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.52.1'
+import os
+
+BASE_VERSION = '0.52.1'
+
+DEFAULT_VERSION_SUFFIX = "-local"
+BUILD_VERSION_FILE_PATH = "./BUILD_VERSION"
+
+
+def _get_build_version():
+    if not os.path.isfile(BUILD_VERSION_FILE_PATH):
+        return None
+
+    with open(BUILD_VERSION_FILE_PATH, 'r') as f:
+        build_version = f.readline()
+
+        return build_version
+
+
+def _get_version():
```

Comment:
missing type annotation `-> str`

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/546/commits/4f30e623a4080384d19f9979ed838bef19432602